### PR TITLE
doc: fix link and few spellings

### PR DESCRIPTION
**Pull request by etienne-lms**

### CODE REVIEW

**Comment by jforissier** [comment]
`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`

File: documentation/secure_storage.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Below is an excerpt from the specification listing the most vital requirements:
    instances of that TA but separated from the other TAs.
 5. The Trusted Storage must provide a minimum level of protection against
    rollback attacks. It is accepted that the actually physical storage may be in
-   an unsecure areas and so is vulnerable to actions from outside of the TEE.
+   an unsecured areas and so is vulnerable to actions from outside of the TEE.
```
Comment (jenswi-linaro) [inline documentation/secure_storage.md]: `s/areas/area/`
`s/unsecured/insecure/`

**Comment by etienne-lms** [comment]
@jforissier : can you plan to integrate this in the 2.4.0 ?

**Comment by jforissier** [comment]
@etienne-lms sure, can you rebase please?

**Comment by etienne-lms** [comment]
my pleasure.

**Comment by jforissier** [comment]
;) merged.

**Pull request merged**